src/java.base/share/classes/java/net/HttpRetryException.java

```diff
@@ -39,7 +39,14 @@
     @java.io.Serial
     private static final long serialVersionUID = -9186022286469111381L;
 
+    /**
+     * The response code.
+     */
     private int responseCode;
+
+    /**
+     * the URL to be redirected to.
```

Comment:
Should this begin with a capital 't' like the previous comment above?

Reply:
Good catch. Fixed.